src/components/Navigation/Header.vue

```diff
@@ -71,9 +77,13 @@
             :small="$vuetify.breakpoint.mdAndDown"
             :x-large="$vuetify.breakpoint.xlOnly"
             class="mr-1 mt-sm-1 teal darken-2"
-            to="/accounts/profile"
```

Comment:
Why did you remove this line? the link to the profile page doesn't work anymore once you are logged in.

Reply:
its been removed accidentally